storybook/storybook-docs/src/typography.stories.mdx

```diff
@@ -1,84 +1,160 @@
 | relatieve regelhoogte      | 1.1 | 1.2 | 1.3 | 1.4 | 1.4 | 1.5 | 1.6 | 1.6 |
 
+Tekst op niveau 5 is groter dan op niveau 4.
+Dat is omdat titel niveau 4 kleinere letters heeft dan paragrafen met grote letters.
+Het alternatief om Heading 4 tekstniveau 5 te geven leek ons nog verwarrender.
+
 Voorbeelden van componenten en hun tekstniveaus:
 
-| level | componenten                          |
-| ----: | :----------------------------------- |
-|     0 | Page heading                         |
-|     1 | Heading 1                            |
-|     2 | Heading 2                            |
-|     3 | Heading 3, Blockquote                |
-|     4 | Heading 4, Top Task Link             |
-|     5 | Paragraph large                      |
-|     6 | Paragraph, alle Links behalve inList |
-|     7 | Paragraph small, inList Link         |
+| Componenten                                          | Niveau |
+| :--------------------------------------------------- | -----: |
+| Page heading                                         |      0 |
+| Heading 1                                            |      1 |
+| Heading 2                                            |      2 |
+| Heading 3, Blockquote                                |      3 |
+| Heading 4, Top Task Link                             |      4 |
+| Large Paragraph, Alert Title                         |      5 |
+| Paragraph, Button, Link behalve InList               |      6 |
+| Small Paragraph, InList Link, Form Label, Breadcrumb |      7 |
 
-## Contrast
+## Accentuering
 
-Het contrast tussen tekst en achtergrondkleur moet voldoen aan [de laatste wettelijke toegankelijkheidseisen van de WCAG-niveau AA](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html).
+Er zijn verschillende manieren om tekst te accentueren.
+Hierin hebben we de volgende keuzes gemaakt.
 
-## Uitlijning
+### Gewicht
 
-Het aantal karakters op een regel is maximaal 80.
-En de tekst wordt altijd links uitgelijnd.
-Dit is vanwege leesbaarheid.
+Online gebruiken we 2 gewichten: ‘regular’ en ‘extra bold’.
 
-Uitzonderingen:
+<Typeset
+  fontFamily="Amsterdam Sans, Arial, sans-serif"
+  fontSizes={[24]}
+  fontWeight={400}
+  sampleText="Jouw typograaf biedt mij zulke exquise schreven"
+/>
+
+Tekst wordt vetgedrukt in titels en decoratieve citaten.
+
+<Typeset
+  fontFamily="Amsterdam Sans, Arial, sans-serif"
+  fontSizes={[24]}
+  fontWeight={800}
+  sampleText="Jouw typograaf biedt mij zulke exquise schreven"
+/>
+
+### Stijl
 
-- Op knoppen worden labels gecentreerd.
-- In tabellen kan de inhoud van kolommen, afhankelijk van de inhoud, ook gecentreerd of rechts uitgelijnd worden.
+Gebruik de cursieve weergave uitsluitend om fragmenten in tekst te accentueren.
+Cursieve tekst is minder goed leesbaar dan reguliere of vetgedrukte tekst.
 
-## Vormgeving
+Pas het niet toe op hele tekstblokken.
+Er zijn betere manieren om lange teksten nadruk te geven.
```

Comment:
Imo, this is also less simple than the original text. How about:

Vermijd cursieve teksten  zo veel mogelijk, deze zijn niet goed te lezen.
Gebruik ze uitsluitend om tekst te accentueren, niet voor hele tekstblokken.
Er zijn betere manieren om lange teksten nadruk te geven.